packages/app/src/components/Root/Root.tsx

```diff
@@ -49,13 +59,218 @@ const SideBarItemWrapper = (props: SidebarItemProps) => {
 
 export const MenuIcon = ({ icon }: { icon: string }) => {
```

Comment:
I would suggest since this component has gotten quite a bit bigger to move this out to it's own file, maybe like [this](https://github.com/janus-idp/backstage-showcase/pull/1557/files#diff-cb937f57126af3528e5f33a39c08f2b89e84310b3351803922558e78c2d0572c) as I found this component useful for the theme selector too in that PR.